vm-image-service/README.md

```diff
@@ -49,14 +27,37 @@ The UI allows users to execute all necessary workflows:
+
+![VM Image List View](source/images/vm-image-image-list.png "VM Image List View")
+In the above picture, we are looking at the list of VirtualMachine Images which are available for the current namespace. Namespaces are not directly associated with VM Images rather transitively associated through Content Libraries. The above list view is an aggregation of all the VM Images across all Content Libraries that are associated with the current namespace.
 
 ### Actions
+![VM Image Delete](source/images/vm-image-delete.png "VM Image Delete")
+The only user action available for consumption users to perform on a VM Image is "Delete" a VM Image from a namespace-scoped Content Library that has been marked as writable. Delete action will be left unavailable for VM Images which belong to cluster-scoped and read-only libraries.
+
 ### Details
+![VM Image Summary](source/images/vm-image-summary.png "VM Image Summary")
+Clicking an image name in the VM Image list view will navigate to a detailed summary view for the selected VM Image. On this view, you will be able to view all the details of VM Image, check health status, and view the list of latest Kubernetes events for the given image.
+
 
-## List all Content Library contents
+## List all Content Library resources
+![Content Library List](source/images/vm-image-library-list.png "Content Library List")
+Content Library list view displays the list of content library resources associated with the current namespace. There are two kinds of library resources, with regards to the scope, *global scoped* and *namespace scoped*. 
+- **Global scoped** library is available for the namespace at the Supervisor level. In other words, global scoped library is common to all the namespaces under a Supervisor cluster. 
+- **Namespace scoped** libraries are associated with a particular namespace by the VI Admin. This kind of library resource is by default **read-only** and can be made **writable**
 
 ### Actions
+*Consumption users do not have the privileges to perform mutation operations on a content library resource*. With the exception of Publishing a VirtualMachine to a writable Content Library.
+
 ### Details
+![Content Library detail](source/images/vm-image-library-details.png "Content Library detail")
+Clicking an library name in the Content Library list view will navigate to a detailed summary view for the selected Content Library resource. On this view, you will be able to view all the details of the Content Library resource, check health status, and view the list of latest Kubernetes events for the given library resource.
 
+## Publish VM to a Content Library
+The overlap between VM Service and VM Image Service is the available VM Publish action on VMs. VMs can be published to a Content Library to be saved them as VM Templates. Below link will provide more information about the VM Publish feature.
```

Comment:
VMs can be published to a Content Library to be saved them as VM Templates. =>
VMs can be published to a Content Library to be saved as VM Templates.